File: src/main/java/com/rkroom/blog/service/CommentService.java
package com.rkroom.blog.service;

import com.rkroom.blog.entity.Comment;

import java.util.List;

public interface CommentService {
    List selectAllByArticleAndPublishstatus(int id, boolean status);
    void insert(Comment comment);
    List selectAllCommentAndItsArticle();
    int publishCommentById(int id);
    int deleteCommentById(int id);
}

File: src/main/java/com/rkroom/blog/service/CommentService.kt
package com.rkroom.blog.service

import com.rkroom.blog.entity.Comment

interface CommentService {
    fun selectAllByArticleAndPublishstatus(id: Int, status: Boolean): List<*>?
    fun insert(comment: Comment)
    fun selectAllCommentAndItsArticle(page: Int): List<*>?
    fun publishCommentById(id: Int): Int
    fun deleteCommentById(id: Int): Int
    fun selectCountAll(): Int
}